sources/sdk/k8s-operator/tests/mocks/kubernetes-client.js

```diff
@@ -2,69 +2,65 @@ const { KubeConfig } = require('@kubernetes/client-node')
 const { withWorld } = require('test!world')
```

Comment:
what is 'test!world'

Reply:
Lock at the `tests/mocks/index.js`, it is a fake module providing a `world` object (mutated with the `withWorld` function).

That way, the test suite can provide callbacks (for the `ResourceWatcher` or the `Operator` classes) and test their results by writing assertions on the `world` object.